lib/use_cases/administrator/get_auth_requests.rb

```diff
@@ -7,10 +7,7 @@ def initialize(authentication_logs_gateway:)
 
       def execute(username: nil, ips: nil)
```

Comment:
I'm wondering if we should rename these keyword args as `username_filter:` and `ip_filter:` to make them more descriptive. It took me a little while to figure out what these were doing as we already have an `ips:` argument for the  gateway. The fact that we now passing in `nil` also confuses it a bit.